edit_topictagged_form.php

```diff
@@ -43,16 +43,10 @@ protected function definition_inner($mform) {
         //remove any that come with the parent class you don't want
 	global $DB, $CFG;
 
-        //Add difficulty field
-        $difficultyoptions = [];
-        $difficultyoptions[0] = 'Easy';
-        $difficultyoptions[1] = 'Easy-Medium';
-        $difficultyoptions[2] = 'Medium';
-        $difficultyoptions[3] = 'Medium-Hard';
-        $difficultyoptions[4] = 'Hard';
-	$difficultyoptions[5] = 'Any difficulty';
-        $mform->addElement('select', 'setdifficulty', get_string('setdifficulty', 'qtype_topictagged'),
-            $difficultyoptions);
+    //Add difficulty field
+    $difficultyoptions = ['Easy', 'Easy-Medium', 'Medium', 'Medium-Hard', 'Hard', 'Any difficulty'];
```

Comment:
This list could be moved to the `consts` file. This will allow easier reuse and make it easier for you to change the categories in the future if required.